datalad/distribution/create_publication_target_sshwebserver.py

```diff
@@ -297,6 +302,11 @@ def __call__(sshurl, target=None, target_dir=None,
                 lgr.error("Failed to add json creation command to post update hook.\n"
                           "Error: %s" % exc_str(e))
 
+            if not only_reconfigure:
+                # Initialize annex repo on remote copy if current_dataset is an AnnexRepo
+                if isinstance(dataset.repo, AnnexRepo):
```

Comment:
On Fri, 09 Sep 2016, Debanjum wrote:

>    In datalad/distribution/create_publication_target_sshwebserver.py:

>  > @@ -297,6 +302,11 @@ def __call__(sshurl, target=None, target_dir=None,
>  >                  lgr.error("Failed to add json creation command to post update hook.\n"
>  >                            "Error: %s" % exc_str(e))

>  > +            if not only_reconfigure:
>  > +                # Initialize annex repo on remote copy if current_dataset is an AnnexRepo
>  > +                if isinstance(dataset.repo, AnnexRepo):

>    this should be datasets[current_dataset].repo instead of dataset.repo

great catch! thanks!